### PR TITLE
docs: how to use ajv8 within fastify3

### PR DESCRIPTION
**Pull request by Eomm**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
   },
   "homepage": "https://www.fastify.io/",
   "devDependencies": {
-    "@fastify/ajv-compiler-8": "github:fastify/ajv-compiler#ajv-8",
+    "@fastify/ajv-compiler-8": "npm:@fastify/ajv-compiler@^2.0.0",
```
Comment (mcollina) [inline package.json:118]: do we need this?

Comment (Eomm) [reply package.json:118]: This alias let us running tests with the ajv8 module, so it lets us spot if the integration is still working.

Before it was pointing to a defined branch in the `fastify/ajv-compiler` project. With the latest release, we can rely on the npm registry

The test in place into `main` is this one:

https://github.com/fastify/fastify/blob/e3903cfaf571dd6c24be3dc2965233869d90e5be/test/schema-special-usage.test.js#L13

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**